style.css

```diff
@@ -56,22 +60,27 @@ body {
 }
 
 #portf {
-  width: 63px;
-  height: 20px;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 600;
-  font-size: 15px;
-  line-height: 20px;
   display: flex;
   align-items: center;
   text-align: center;
-  color: #fff;
   flex: none;
   order: 0;
   flex-grow: 0;
 }
 
+.desktop_menu_link {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.desktop_menu_link:hover {
+  color: #f55800;
+}
+
```

Comment:
- [ ] Kindly check the comments on the picture in order to match better the given design

Yours:
![image](https://user-images.githubusercontent.com/62705947/208995661-8d6b708d-ac1c-4f38-b116-1492cf967542.png)
Design:
![image](https://user-images.githubusercontent.com/62705947/208995718-c6067e93-3744-4fc4-8d49-413587a5b95d.png)

Reply:
![image](https://user-images.githubusercontent.com/62705947/208996086-ff562854-7d89-486c-81b2-4168b454bb10.png)

Reply:
- [ ] Check the requirement for mobile:
![image](https://user-images.githubusercontent.com/62705947/208996543-f0f52018-9689-4c66-81c6-df9da487193d.png)

Reply:
![image](https://user-images.githubusercontent.com/62705947/208996884-f636e647-e201-468b-9262-ed2102d18acb.png)

Reply:
Where do you want it to lead to? no where?

Reply:
Hopefully I fixed it as desired, made it a pop up - but information here is lacking

Reply:
Besides a few kinks and missing my request for clarification, this was a surprisingly well done code review 👍👍👍

Please accept:
🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏